NetworkTableViewer/2018NetworkTableViewer/src/org/snobot/nt/dotstar/DotstarSimulatorPanel.java

```diff
@@ -11,6 +11,7 @@
 public class DotstarSimulatorPanel extends JPanel
```

Comment:
Updated the simulation widget to have two side by side widgets of 72 lights rather than one long horizontal strip of 144 lights.  Better represents the setup on the robot.  There is a potential that the LEDs are not in the "right order"